shared/nav.native.js

```diff
@@ -123,14 +123,6 @@ function MainNavStack (props: Props) {
   return <Container hideNav={props.hideNav} shim={shim} tabBar={tabBar} />
 }
 
-function renderFullScreenStackRoute (route) {
```

Comment:
I'm not sure there was a reason -- I think whoever made the changes (me?) just didn't think through that they had the same rendering requirements.